src/see_back.py

```diff
@@ -0,0 +1,12 @@
+"""Даны два натуральных числа.
+Вычислите их наибольший общий делитель при помощи
+алгоритма Евклида (мы не знаем функции и рекурсию)."""
+
+a = int(input())
+b = int(input())
+while a != 0 and b != 0:
+    if a > b:
+        a = a % b
+    else:
+        b = b % a
+print(a + b)
```

Comment:
it's pretty tricky - in a such case you try to handle 0 in one of your variables. according to algorithm GCD is not a sum.